simulators/totalpower/__init__.py

```diff
@@ -101,7 +102,7 @@ def parse(self, byte):
 
     def _execute(self, msg):
         msg = msg.strip()
-        print msg
+        print(msg)
```

Comment:
Prospector 1.2.0 gives me the following error regarding this line:
pylint: superfluous-parens / Unnecessary parens after 'print' keyword